packages/dotcom-build-js/README.md

```diff
@@ -1,11 +1,9 @@
 # @financial-times/dotcom-build-js
```

Comment:
It would be good to document the use of `babel` in this plugin as it's been moved out of the base styles package now and it's a reasonably important piece of the Page Kit puzzle.

Reply:
just to be clear, do you mean reinstate some documentation that's missing from this branch, or write new documentation for babel? the only babel docs i can see that have been remove are around the hooks for customising the config, which aren't applicable any more

Reply:
Let's not worry about adding new documentation but an indication in this readme that this is where the babel happens would probably be useful. Babel docs are pretty accessible if folks need extra context.